app/scripts/controllers/specialErrorDetail.js

```diff
@@ -26,23 +28,48 @@ module.exports = /*@ngInject*/ function ($scope, $routeParams, $location, $http,
         $scope.error = {};
     }
 
+    $scope.sort = function(property) {
+        $scope.reverse = $scope.sortedBy === property ? !$scope.reverse : false;
```

Comment:
I named it `reverse` because that's what Angular calls it in [their `orderBy` filter](https://docs.angularjs.org/api/ng/filter/orderBy). I can rename it to something like `sortAsc` if you think that makes more sense.